src/main/java/seedu/address/model/order/Order.java

```diff
@@ -77,6 +77,10 @@ public Complete getComplete() {
         return complete;
```

Comment:
If you have the isComplete() method, then what is the value of this method?

Reply:
The getComplete method was already implemented in the code before this feature, it returns the Complete object. isComplete returns the boolean value of the Complete object. For my purposes in this feature, I needed the boolean value and not the Complete object, but accessing the Complete object to get the boolean, from a separate class where I needed the value, violates the law of demeter. Therefore, I needed an isComplete method.